src/WebJobs.Extensions.DurableTask/ContextImplementations/DurableClient.cs

```diff
@@ -953,6 +983,7 @@ private static void AddScheduledEventDataAndAggregate(ref Dictionary<string, Eve
             {
                 historyItem["ScheduledTime"] = taskScheduledData.Date;
                 historyItem["FunctionName"] = taskScheduledData.Name;
+                historyItem["Input"] = taskScheduledData.Input;
```

Comment:
With this change, you're always adding this input value but we should only display this when the customer is using this new showHistoryInput setting.